### ISSUE TITLE
WindowsError: [Error 2]  when trying to do npm install

### ISSUE THREAD

**Issue by ghost** [LLK/scratch-blocks#2060]
### Expected Behavior

Forked and cloned this repo to my PC. I followed the guide in [this wiki](https://github.com/LLK/scratch-gui/wiki/Getting-Started#slightly-different-for-scratch-blocks). Expected to install 

### Actual Behavior

The following text appear in Git Bash when I typed in `npm i` in the directory of the clone
```bash
Traceback (most recent call last):
  File "build.py", line 574, in <module>
    test_proc = subprocess.Popen(test_args, stdin=subprocess.PIPE, stdout=subprocess.PIPE)
  File "C:\Python27\lib\subprocess.py", line 394, in __init__
    errread, errwrite)
  File "C:\Python27\lib\subprocess.py", line 644, in _execute_child
    startupinfo)
WindowsError: [Error 2] 
```
### Steps to Reproduce

clone the repo and follow the steps [here](https://github.com/LLK/scratch-gui/wiki/Getting-Started#slightly-different-for-scratch-blocks)
When you typed `npm i` in bash OR cmd, you should see the error above appear and the process fail.

### Operating System and Browser

Window 10, Python 2.7.17 64-bit

**Comment by cwillisf** [comment]
We should fix this, but in the mean time I recommend using Cygwin or WSL to work on scratch-blocks in Windows. WSL is more reliable and is the way I do most of my work these days.

WSL setup instructions (requires Windows 10): https://docs.microsoft.com/en-us/windows/wsl/install-win10
Cygwin setup instructions: https://cygwin.com/install.html

Note that you'll need to install Python "inside" of WSL or Cygwin for this to be a complete & reliable solution. Once you have everything installed, the regular `npm i` should work from the WSL or Cygwin prompt.

**Comment by cwillisf** [comment]
This specific issue is resolved but #2080 still blocks scratch-blocks development on plain Windows.
